### PR TITLE
Improve coverage and fix CI

### PR DESCRIPTION
**Pull request by jtakalai**
This should only be reviewed and merged after #106

### CODE REVIEW

File: packages/data-union-solidity/contracts/DataUnionSidechain.sol [hunk 1]
```diff
@@ -134,20 +134,18 @@ contract DataUnionSidechain is Ownable, IERC20Receiver, IERC677Receiver {
...
     function setFees(uint256 newAdminFee, uint256 newDataUnionFee) public onlyOwner {
-        require((newAdminFee + newDataUnionFee) <= 1 ether, "error_Fees");
+        require((newAdminFee + newDataUnionFee) <= 1 ether, "error_fees");
         adminFeeFraction = newAdminFee;
         dataUnionFeeFraction = newDataUnionFee;
         emit FeesSet(adminFeeFraction, dataUnionFeeFraction);
     }
 
-    function setDataUnionBeneficiary(address _dataUnionBeneficiary) public onlyOwner {
-        require(_dataUnionBeneficiary != address(0), "invalid_address");
```
Comment (samt1803) [inline packages/data-union-solidity/contracts/DataUnionSidechain.sol:144]: why was this removed? what happens if you use 0 as address?

**Review by samt1803 — APPROVED** [review]
looks good!

File: packages/data-union-solidity/contracts/DataUnionSidechain.sol (hunk 1 above)
Comment (jtakalai) [reply packages/data-union-solidity/contracts/DataUnionSidechain.sol:144]: Then the beneficiary gets set to zero. There should be no special logic in place for zero beneficiaries (indirectly maybe if member 0x0000 were special, but it shouldn't be). This is onlyOwner function, and I see no reason zero were different from just random dummy address.
